File: legacy/src/main/java/com/redbox/domain/funding/dto/FundingFilter.java
package com.redbox.domain.funding.dto;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;

@Setter
@Getter
public class FundingFilter {
    private int page;
    private int size;
    private Filter sort;
    private Filter option;
    private LocalDate startDate;
    private LocalDate endDate;
}

File: migration/src/main/kotlin/com/redbox/domain/community/funding/dto/FundingFilter.kt
package com.redbox.domain.community.funding.dto

import java.time.LocalDate

class FundingFilter {
    val page = 0
    val size = 0
    val sort: Filter? = null
    val option: Filter? = null
    val startDate: LocalDate? = null
    val endDate: LocalDate? = null
}
